css/templates/e-commerce/page.css

```diff
@@ -202,16 +412,78 @@ body {
     background-color: var(--color-white);
     border: none;
     color: var(--color-primary);
-    font-size: var(--h5);
+    font-size: var(--h7);
   }
 
   .commerce-hero-content-theme-secondary,
   .commerce-hero-content-theme-secondary .commerce-hero-content-title {
-    color: var(--color-secondary);
+    color: var(--color-primary);
   }
 
   .commerce-hero-content-theme-secondary .ampstart-btn {
     color: var(--color-white);
-    background-color: var(--color-secondary);
+    background-color: var(--color-primary);
+  }
+}
+
+
+/* PRODUCT DETAILS */
+.icon-star,
+.icon-star-empty {
+  height: 14px;
+  width: 14px;
+  color: var(--color-font-accent);
+}
+
+.icon-star-empty {
+  color: #dadada;
+}
+
+.commerce-product-btn-wrapper {
+  text-align: center;
+}
+
+@media (--breakpoint-md) {
+  .commerce-product-btn-wrapper {
+    text-align: initial;
+  }
+}
+
+.commerce-product-color-swatch {
+  height: 24px;
+  width: 24px;
+  margin-left: var(--space-1);
+}
+
+.commerce-product-color-swatch:first-child {
+  margin-left: 0;
+}
+
+.commerce-product-color-white {
+  background-color: var(--color-white);
+}
+
+.commerce-product-color-green {
+  background-color: #243d1e;
+}
+
+.commerce-product-thumb {
+  width: 43px;
+  margin-left: var(--space-1);
+}
+
+.commerce-product-thumb:first-child {
+  margin-left: 0;
+}
+
+@media (--breakpoint-md) {
+  .commerce-product-thumb {
+    width: 80px;
+  }
+}
+
+@media (--breakpoint-md) {
+  .commerce-product-desc {
```

Comment:
I did, the spacing is only required on the right and it should be the width of one column (8.3%), however the gutters are based on rem units in whole numbers and applied to both sides. Maybe I could make the width of col-1 into a css var and then use that?